client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -63,23 +85,6 @@ const Launchpad: Step = ( { navigation, flow }: LaunchpadProps ) => {
 		}
 	}, [ verifiedParam, translate, dispatch ] );
 
-	useEffect( () => {
```

Comment:
Best I can tell, none of the logic here needs to be in a useEffect, so I've removed it and shifted it above. In addition to removing the unneeded useEffect, this also has the effect of triggering the code faster on the frontend. There's still a flicker of the launchpad before redirecting, but it's much faster without the useEffect.